contracts/WildcardSteward_v2.sol

```diff
@@ -48,6 +48,9 @@ contract WildcardSteward_v2 is Initializable {
 
     MintManager_v2 public mintManager;
 
+    mapping(uint256 => address) artistAddresses; //mapping from tokenID to the artists address
+    mapping(uint256 => uint256) wildcardsPercentages; // mapping from tokenID to the percentage sale cut of wildcards for each token
```

Comment:
For every wildcards token, this keeps track of the %age commission we will receive. This way, in advanced settings on the front end as I see it, the user can choose to give a %age commission on the sale of their wildcards. It will be a minimum of 5% as currently implemented. We should decide on the front end the amount we think should be default (say 10%) ? 

We should also include some events, and award achievements etc.. to users who make this percentage higher in support of our platform.

Reply:
Actually, it can be a bit confusing for people buying; they are actually setting the commission they will give us when someone buys it off them, not the commission of the current purchase. We must just make this clear. (This is what you had in mind right?)

It doesn't make sense otherwise, because then I would always set the commission to the highest possible because I don't really care about the previous person.

I think it could be really awesome to allow people to redirect commissions towards one of the DAOs? Not for this iteration for sure!

Reply:
Yeah you are totally right that they are setting the commission for when its bought off them, its implemented like that. Otherwise you are right it would make no sense :)

Yeah the DAO idea is awesome. Redirect toward the monthly vote :) definitely next scope